README.md

```diff
@@ -31,19 +27,19 @@ From the `./test` directory make all of the regression images with:
     $ make images
 
 *`make -j images` can be used to make the images in parallel (faster), although this may lead to
-Docker Desktop*
+Docker Desktop instabilities*
```

Comment:
Interesting. I thought this line used to include this. Ah well. Good catch!

Reply:
It did and I noticed it was missing. /shrug